appveyor.yml

```diff
@@ -2,3 +2,25 @@ configuration: Release
 test:
   assemblies:
     - Argu.Tests.dll
+
+install:
+  - ps: $env:CLI_VERSION = "1.0.0-preview2-003121"
+  - ps: $env:DOTNET_INSTALL_DIR = "$pwd\.dotnetcli"
+  # Download install script to install .NET cli in .dotnet dir

```

Comment:
Thanks for these hints. I tried to make the official "obtain" script working, but its broken. Same for the Link within the obtain script. Is it working again or how did you get the link (which is working) in the linked example. Anyway thanks, will change it...